data.sql

```diff
@@ -11,4 +11,35 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 ('Angemon', '2005-06-12', 1, true, -45),
 ('Boarmon', '2005-06-07', 7, true, 20.4),
 ('Blossom', '1998-09-13', 3, true, 17),
-('Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+('Ditto', '2022-05-14', 4, true, 22);
+
+
+INSERT INTO owners (full_name, age) VALUES('Sam Smith', 34),('Jennifer Orwell', 19),('Bob', 45),('Melody Pond', 77),('Dean Winchester', 14),('Jodie Whittaker', 38);
+
+INSERT INTO species(name) Values('Pokemon');
+INSERT INTO species(name) Values('Digimon');
+
+-- If the name ends in "mon" it will be Digimon
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+-- All other animals are Pokemon
+UPDATE animals SET species_id = 1 WHERE name LIKE '%mon';
```

Comment:
- [ ] Both of them are the same, doesn't it? Please 🙏🏽 modify your inserted animals so it includes the species_id value for Pokemon, it could be something like this 👇🏽 

```
UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;
```

Reply:
![image](https://user-images.githubusercontent.com/109465466/230681243-70dd5921-c382-49e1-bdd9-19dea7319663.png)
Hello, made changes